usr/src/cmd/bhyve/block_if.c

```diff
@@ -630,6 +650,10 @@ blockif_open(const char *optstr, const char *ident)
 				}
 			}
 		}
+
+		if (nodelete == 0 && ioctl(fd, DKIOC_CANFREE, &candelete))
```

Comment:
`ioctl()` does not return a `bool` or `boolean_t`, so its return value should be compared to 0 or -1.

Reply:
The extant freebsd code uses this style, so I opted for consistency with it (though I agree that being explicit would be better). We don't upstream the stuff in `#ifndef __FreeBSD__` blocks AFAIK, so there's no upstream issue though that would cause conflicts there, so I don't feel too strongly either way.

Reply:
I guess this isn't the only place where I hold my nose in the bhyve code.